### PR TITLE
Home page

### PR DESCRIPTION
**Pull request by Muzammil-khan710**
Home page

Note: links are not have been updated yet. I will fix them when i will be adding react router.

### CODE REVIEW

File: src/Components/Homepage/Homepage.jsx [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+        <img style={{height:"auto",width:"100%"}} src={HomeImg} alt="" />
+        <section className='section-area'>
+            <h2 className='section-text'>Exclusive offer upto 50% off</h2>
+            <button className='explore-btn'>Explore all categories</button>
+        </section>
+        <h1>Available Categories</h1>
+        <section className='category-container'>
+        <div className="featured-product-type">
+          <FeaturedProducts src={AutomaticWatch} title="AutomaticWatch" />
+          <FeaturedProducts src={AnalogWatch} title="AnalogWatch" />
```
Comment (rutu14) [inline src/Components/Homepage/Homepage.jsx:18]: Suggestion: Can create a array of data values and reuse them using map function.

**Review by rutu14 — APPROVED** [review]
The code is good to merge.

File: src/Components/Homepage/Homepage.jsx (hunk 1 above)
Comment (Muzammil-khan710) [reply src/Components/Homepage/Homepage.jsx:18]: I haven't added data yet. i Will be implementing when i will add data for products. Thanks for the suggestion !

**Review by archit7448 — APPROVED** [review]
You can add data for category  the map , otherwise code looks good to merge

**Comment by Muzammil-khan710** [comment]
> You can add data for category the map , otherwise code looks good to merge

I will add data when i am working on product page
Thank You

**Pull request merged**